apps/material-react-table-docs/components/prop-tables/tableOptions.ts

```diff
@@ -357,6 +357,17 @@ export const tableOptions: TableOption[] = [
     source: 'MRT',
     type: 'boolean',
   },
+  {
+    tableOption: 'enableGroupingSingleColumn',
```

Comment:
Could we instead just use the already existing `groupedColumnMode` option instead of adding this for the logic of whether to show a single column? Maybe not.

Reply:
I think we can use it.